### ISSUE TITLE
Notebook - Kusto: change connection is stuck

### ISSUE THREAD

**Issue by MsSQLGirl** [microsoft/azuredatastudio#12384]
Azure Data Studio Version:
Version: 1.22.0-insider (user setup)
Commit: 7e76f8c
Date: 2020-09-16T05:43:11.572Z
VS Code: 1.48.0
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041

Steps to Reproduce:

1. Create a connection with ADX cluster with an alias. Do not specify a database name.
2. On the Connections viewlet, right click on an ADX database.
3. Click on New Notebook.
You'll see that the notebook is launched but connecting to the the ADX cluster from the Attach To column. Note that the notebook is actually really connected to the ADX database. 
4. Click on Change Connection.
5. Update the existing connection such that the database (from Default) is pointing to Samples. 
6. Click Connect.

Actual: 
After clicking Connect, you'll see something like this: 
![image](https://user-images.githubusercontent.com/6405919/93401883-2a508b00-f838-11ea-8193-277e76100ac8.png)

Since it looks like it wasn't really connecting, user might decide to re-enter the details. See below. 
![KustoNotebookChangeConnection](https://user-images.githubusercontent.com/6405919/93401933-448a6900-f838-11ea-957f-41dad3961d78.gif)

Expected:
Close the dialog and connection should be updated.

**Comment by MsSQLGirl** [comment]
@VasuBhog , @gupta1 : FYI.

**Comment by VasuBhog** [comment]
This also happens for SQL connections as well (if connected to SQL connection with an **alias**).

**Comment by chlafreniere** [comment]
@MsSQLGirl @VasuBhog is this a regression from August?

**Comment by VasuBhog** [comment]
@chlafreniere No, this issue was also present in August (for SQL connections with an alias).

**Comment by chlafreniere** [comment]
Given that this isn't a regression, moving out of September.

**Comment by MsSQLGirl** [comment]
Confirmed. Not a regression and also happening for SQL. Thanks!
